docs/Install.md

```diff
@@ -14,7 +14,39 @@ On Linux the easiest way to install JackTrip is to use the distribution's packag
     sudo apt install jacktrip
     ```
 
-The [GitHub releases page](https://github.com/jacktrip/jacktrip/releases) also includes a binary which should run on most Linux distributions (x64). This build is known to not look well under Wayland.
+### Latest release
+
+If your distribution doesn't include the latest release in their repository, there are different
+alternative options to install from.
+
+=== "Ubuntu"
+
+    Ubuntu users can install from a [PPA repository](https://launchpad.net/~umlaeute/+archive/ubuntu/jacktrip/)
+    maintained by the packager of Ubuntu's official JackTrip packages.
+
+    ```bash
+    sudo add-apt-repository ppa:umlaeute/jacktrip
+    sudo apt update
+    sudo apt install jacktrip
+    ```
+
+=== "Debian"
+
+    Debian stable users can obtain their latest version from the [Debian backports] repository.
```

Comment:
IMO square brackets in `[Debian backports]` look a little weird. It can stay as is, but I think it might look better as one of these:
"Debian Backports"
Debian "Backports"
`Debian Backports`

Reply:
That should have been a link.

Reply:
Fixed